frontend/app/forms/motion-to-waive-fees/motion-to-waive-fees.web.component.js

```diff
@@ -81,10 +81,41 @@ const debtOwed = [
   "MinimumMonthlyPayment"
 ];
 
+const bankOrCredit = [
+  "AccountNumber",
+  "DateOpened",
+  "NameAndAddress",
+  "NamesOnAccount",
+  "CurrentBalance"
+];
+
+const stocksAndSecurities = [
+  "AccountNumber",
+  "DateOpened",
+  "NameAndAddress",
+  "NamesOnAccount",
+  "CurrentBalance"
+];
+
+const moneyOwedToMe = [
+  "DateOfLoan",
+  "NameAndAddress",
+  "NamesOnAccount",
+  "CurrentBalance"
+];
+
+const otherAssets = [
+  "AccountNumber",
+  "DateOpened",
+  "NameAndAddress",
+  "NamesOnAccount",
+  "CurrentBalance"
+];
+
 export default function MotionToWaiveFees_Web({ data }) {
   const scope = useCss(css);
   return (
-    <FormThatPrints {...css}>
+    <FormThatPrints {...scope}>
```

Comment:
This won't work. You have to put kremling's css scope on a dom element, not a react component. See the section "Can I apply a css scope to a React component (instead of a dom element)?" at https://kremling.js.org/api/use-css.html